src/shared/sam/localLambdaRunner.ts

```diff
@@ -3,8 +3,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { copyFile, unlink, writeFile } from 'fs-extra'
+import { copyFile, unlink, readFile, writeFile } from 'fs-extra'
 import * as path from 'path'
+import * as request from 'request'
```

Comment:
Can you merge this with the `httpResourceFetcher`?

Reply:
Is the risk worth it right now?